aries-site/src/pages/foundation/tshirt-sizing.mdx

```diff
@@ -32,7 +34,9 @@ Finally, t-shirt sizes enable content to responsively scale more easily and cons
 
 In order to create the range of sizes and their corresponding pixel values, a base unit of `24px` is used as a scalar.
 
-Below is a snippet from Grommet's [`base.js`](https://github.com/grommet/grommet/blob/master/src/js/themes/base.js) theme file where the calculations and resulting pixel values for the t-shirt sizes driving the height/width for components such as [Box](/components/box) are defined. The `base.js` theme establishes many of the defaults leveraged across Grommet applications.
+### Original scale (v6 and below)
```

Comment:
I'm wondering if we just talk about the v7 t-shirt sizes on this page and have a link that takes users to a separate page if they need info on the old t-shirt size scale?